labelme/widgets/label_dialog.py

```diff
@@ -146,4 +212,4 @@ def popUp(self, text=None, move=True):
         self.edit.setFocus(QtCore.Qt.PopupFocusReason)
         if move:
             self.move(QtGui.QCursor.pos())
-        return self.edit.text() if self.exec_() else None
+        return (self.edit.text(), self.getFlags() if self.flags else None) if self.exec_() else (None, None)
```

Comment:
Can you make this more human-friendly?

Reply:
By breaking it up into an if/else statement other than a 1 liner?

Reply:
yes.